openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceEntity.spec.ts

```diff
@@ -124,6 +123,10 @@ entities.forEach((EntityClass) => {
         // increase timeout as it using single test for multiple steps
         test.slow(true);
 
+        const token = await getToken(page);
+        const apiContext = await getAuthContext(token);
```

Comment:
@ShaileshParmar11 , I think we can have one until which can return the API context directly so we don't have to repeat that get token line.